src/include/duckdb/common/dl.hpp

```diff
@@ -32,13 +32,13 @@ void *dlsym(void *handle, const char *name) {
 	return (void *)GetProcAddress((HINSTANCE)handle, name);
```

Comment:
I suppose these should also be internal?

Reply:
I haven't really tried to build this on Windows, but I suppose you're right. Let me amend.